jerry-core/jrt/jrt.h

```diff
@@ -86,9 +86,9 @@ extern void __noreturn jerry_unimplemented (const char *, const char *, const ch
 #define JERRY_LOG(lvl, ...) \
   do \
   { \
-    if (lvl <= jerry_debug_level && jerry_log_file) \
+    if (lvl <= jerry_debug_level) \
```

Comment:
@akosthekiss, as I said. I'll do it in a follow up work. Don't worry. It's on my list to clean this up.